src/style.css

```
@@ -0,0 +1,131 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 0 0 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  width: 100%;
+  font-weight: 500;
+  font-size: 12px;
+  margin-bottom: 4px;
+  line-height: 18px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  align-self: flex-start;
+  margin: 0 0 16px;
+  width: 100%;
+  line-height: 14px;
+}
+
+.card__stars-review {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.card__stars-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 24px;
+  width: 100%;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-star {
+  background-image: url(images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 div:nth-of-type(n + 1),
+.stars--1 div:nth-of-type(n + 2),
+.stars--2 div:nth-of-type(n + 3),
+.stars--3 div:nth-of-type(n + 4),
+.stars--4 div:nth-of-type(n + 5),
+.stars--5 div:nth-of-type(n + 6) {
+  background-image: url(images/star.svg);
+}
+
+.card__price-container {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  line-height: 18px;
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: 16px;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__button[data-qa="hover"]:hover {
+  background-color: white;
+  color: #00acda;
+}
```

`data-qa="hover"` needs only for testing. Your selector should looks like this
```suggestion
.card__button:hover {
  background-color: white;
  color: #00acda;
}
```